message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -309,6 +309,12 @@
       <entry value="1" name="SCRIPTING_CMD_REPL_STOP">
         <description>End a REPL session.</description>
       </entry>
+      <entry value="2" name="SCRIPTING_CMD_STOP">
```

Comment:
If it were me I would add START, STOP and RESTART.  I think most users will understand restarting involves stopping the script if it is already running so no need for the "STOP_AND" part I think.

Reply:
I didn't add start because is the same as stop and restart if your already stopped. You can't really start without stopping, for example you can't have scripting turned off at boot and enable it as some later point and then start. 

I guess we could just call it "start" and not "stop and restart". 

No real objection to adding start tho, I just don't think anyone would use it. You either want to stop a rouge script or restart a script that has crashed. I also wanted to be clear that this is all scripts you can if one has crashed and one still going this method will stop and restart both.

Reply:
Yeah, I guess you're right that there's no point in having "start" when we have "restart".  I'd just call them "stop" and "restart"... but I don't feel strongly about it and this is your baby.